### PR TITLE
Add tests for collection removal without triggering validation

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Changed
- Bump `sonata-project/form-extensions` to ^1.19
```

### CODE REVIEW

**Comment by jordisala1991** [comment]
Once the PR is merged and released, this should not fail (It will require bump requirements to not fail on lower deps @VincentLanglet

**Comment by jordisala1991** [comment]
With this we have tests for the CollectionType, case where we fail validation (not removing an item) and case where the validation is not executing because even it the collection item is not valid, it is also removed.

File: composer.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
         "doctrine/persistence": "^3.0.2",
         "sonata-project/admin-bundle": "^4.18",
         "sonata-project/exporter": "^2.0 || ^3.0",
-        "sonata-project/form-extensions": "^1.4",
+        "sonata-project/form-extensions": "1.x-dev",
```
Comment (jordisala1991) [inline composer.json]:
```suggestion
        "sonata-project/form-extensions": "^1.19",
```

**Review by VincentLanglet — APPROVED** [review]

**Comment by VincentLanglet** [comment]
I would consider bumping a dependency minor as pedantic, no ?

**Comment by jordisala1991** [comment]
Not sure about that TBH , first I tagged as pedantic, but since it can affect users of this package, I changed it to minor.

**Comment by VincentLanglet** [comment]
> Not sure about that TBH , first I tagged as pedantic, but since it can affect users of this package, I changed it to minor.

Developper doesn't need any notification since composer will do the version resolution.
And it would be better to tell dev-kit that no release is needed so far for this library.

**Pull request merged**